assets/js/services/global-vars.js

```diff
@@ -7,3 +7,70 @@ export const IIIF_SIZES = {
   IIIF_SQUARE: "/square/500,500/0/default.jpg",
   IIIF_FULL: "/full/full/0/default.jpg"
 };
+export const PRESERVATION_LEVELS = [
+  { label: "Level 1", id: 1 },
+  { label: "Level 2", id: 2 },
+  { label: "Level 3", id: 3 }
+];
+export const RIGHTS_STATEMENTS = [
+  {
+    id: " http://rightsstatements.org/vocab/InC/1.0/",
+    term: "In Copyright",
```

Comment:
For consistency, can you replace the key "term" with "label" in these objects, so it follows suit with `PRESERVATION_LEVELS` and other helpers we may use?